omod/src/main/webapp/fragments/editresults.gsp

```diff
@@ -0,0 +1,192 @@
+<% ui.includeJavascript("laboratoryapp", "jQuery.print.js") %>
+
+<script>
+    jq(function(){
+        jq('#date').datepicker("option", "dateFormat", "dd/mm/yy");
+
+        jq('#get-results').on('click', function () {
+            var date = moment(jq('#accepted-date-field').val()).format('DD/MM/YYYY');
+            var phrase = jq("#phrase").val();
+            var investigation = jq("#investigation").val();
+            jq.getJSON('${ui.actionLink("laboratoryapp", "results", "searchForResults")}',
+                    {
+                        "date" : date,
+                        "phrase" : phrase,
+                        "investigation" : investigation
+                    }
+            ).success(function(data) {
+                        if (data.length === 0) {
+                            jq().toastmessage('showNoticeToast', "No match found!");
+                        }
+                        result.items.removeAll();
+                        jq.each(data, function(index, testInfo){
+                            result.items.push(testInfo);
+                        });
+                    });
+        });
+    });
+</script>
+
+<div>
+    <form>
+        <fieldset>
+            ${ui.includeFragment("uicommons", "field/datetimepicker", [id: 'accepted-date', label: 'Date', formFieldName: 'acceptedDate', useTime: false, defaultToday: true])}
+            <label for="phrase">Patient Identifier/Name</label>
+            <input id="phrase"/>
```

Comment:
We need to come up with a different id for this input. I'm sure it's the same in the `Queue` and `Worklist`. How about we change them to: `search-results-for`, `search-queue-for` and `search-worklist-for`. This will affect the labels and the corresponding javascript scripts.